src/PhpSpreadsheet/Style/NumberFormat/PercentageFormatter.php

```diff
@@ -9,8 +9,8 @@ class PercentageFormatter extends BaseFormatter
     /** @param float|int $value */
     public static function format($value, string $format): string
     {
-        if ($format === NumberFormat::FORMAT_PERCENTAGE) {
-            return round((100 * $value), 0) . '%';
+        if (NumberFormat::FORMAT_PERCENTAGE === $format) {
+            return round((100 * $value)) . '%';
```

Comment:
I don't believe that either of these statements needs to change. If you think that they do, please let me know why and provide an example. Otherwise, please revert these statements to the original to make the "meat" of this change a bit cleaner.

Reply:
No it's just auto formatting I have on, reverted the lines.